numpy/lib/arraysetops.py

```diff
@@ -380,106 +381,130 @@ def setxor1d(ar1, ar2, assume_unique=False):
     flag2 = flag[1:] == flag[:-1]
     return aux[flag2]
 
-def in1d(ar1, ar2, assume_unique=False, invert=False):
+def isin(elements, test_elements, assume_unique=False, invert=False):
```

Comment:
It's not worth the hassle of changing argument names on an effectively deprecated function and potentially breaking some users. Let's only use the new arguments names for `isin`.